qemu/tests/qemu_guest_agent.py

```diff
@@ -1873,6 +1873,11 @@ def _change_bl(self, session):
         """
 
         if self.params.get("os_type") == "linux":
+            error_context.context(
+                "Set selinux policy to 'Permissive' mode in guest.", LOG_JOB.info
+            )
+            if session.cmd_output("getenforce").strip() != "Permissive":
```

Comment:
When the SELinux state is `Disabled`, this step will fail.

Reply:
ah, that's a point. but how do you usually deal with SELinux==disabled situation? add condition:

```python
if session.cmd_output("getenforce").strip() == "Disbaled":
    pass
``` 
or just add a command above `# don't include situation that selinux is disabled.`

Reply:
Only when state == `Enforcing` we should disable it, so simplify to `("getenforce").strip() == "Enforcing"`

Reply:
okay. will update it.

Reply:
updated.